frontend/src/pages/AccountSend.vue

```diff
@@ -101,12 +101,21 @@
         :rules="isValidId"
         ref="recipientIdBaseInputRef"
       />
+      <div
+        class="flex q-pb-lg q-pt-sm text-caption warning-container"
+        v-if="recipientIdWarning"
+        :style="!recipientId || isValidRecipientId ? 'margin-top:-2em' : ''"
```

Comment:
This may be a good candidate to address in a separate issue as this experience exists in the current app. I reproduced by copying .eth into the input and clicking away.

![Screenshot from 2023-05-31 15-06-47](https://github.com/ScopeLift/umbra-protocol/assets/12705146/c0293385-ed6a-4324-a44a-489e65202a57)

Reply:
Heh that one also doesn't look great, yea I'm ok with creating a separate issue to track/resolve both of the issues in the above images. Feel free to resolve once we have the issue

Reply:
https://github.com/ScopeLift/umbra-protocol/issues/525